roles/ansible_service_broker/tasks/install.yml

```diff
@@ -156,6 +175,34 @@
             app: openshift-ansible-service-broker
             service: asb
 
+- name: create asb-etcd service
+  oc_obj:
+    name: asb-etcd
+    namespace: openshift-ansible-service-broker
+    state: present
+    kind: Service
+    content:
+      path: /tmp/asbetcdsvcout
```

Comment:
Is it possible to use a dictionary here instead of hard-coding all this into the task?